install_config/install/disconnected_install.adoc

```diff
@@ -0,0 +1,550 @@
+    registry.access.redhat.com/openshift3/ose-pod \
+    registry.access.redhat.com/openshift3/ose-docker-registry 
+----
+
+. If you synchronized the metrics and log aggregation images, export:
++
+----
+# docker save -o ose3-logging-metrics-images.tar \
+    registry.access.redhat.com/openshift3/logging-deployment \
+    registry.access.redhat.com/openshift3/logging-elasticsearch \
+    registry.access.redhat.com/openshift3/logging-kibana \
+    registry.access.redhat.com/openshift3/logging-fluentd \
+    registry.access.redhat.com/openshift3/logging-auth-proxy \
+    registry.access.redhat.com/openshift3/metrics-deployer \
+    registry.access.redhat.com/openshift3/metrics-hawkular-metrics \
+    registry.access.redhat.com/openshift3/metrics-cassandra \
+    registry.access.redhat.com/openshift3/metrics-heapster
+----
+
+. Export the S2I builder images, making sure to remove any items that you did not sync in the previous section:
++
+----
+# docker save -o ose3-builder-images.tar \
+    registry.access.redhat.com/jboss-amq-6/amq-openshift \
+    registry.access.redhat.com/jboss-eap-6/eap-openshift \
+    registry.access.redhat.com/jboss-webserver-3/tomcat7-openshift \
+    registry.access.redhat.com/jboss-webserver-3/tomcat8-openshift \
+    registry.access.redhat.com/rhscl/mongodb-26-rhel7 \
+    registry.access.redhat.com/rhscl/mysql-56-rhel7 \
+    registry.access.redhat.com/rhscl/perl-520-rhel7 \
+    registry.access.redhat.com/rhscl/php-56-rhel7 \
+    registry.access.redhat.com/rhscl/postgresql-94-rhel7 \
+    registry.access.redhat.com/rhscl/python-27-rhel7 \
+    registry.access.redhat.com/rhscl/python-34-rhel7 \
+    registry.access.redhat.com/rhscl/ruby-22-rhel7 \
+    registry.access.redhat.com/openshift3/nodejs-010-rhel7 
+---- 
+
+[[disconnected-repo-server]]
+== Repository Server
+
+During the installation (and for later updates, should you so choose) you will
+need a webserver to host the repositories. RHEL7 can provide the Apache
+webserver.
+
+*Option 1*: Re-configure as a Web server
+
+If you can re-connect the server where you synchronized the software and images
+to your LAN, then you can simply install Apache on the server:
+
+----
+# yum install httpd
+----
+
+Skip to link:#disconnected-placing-the-software[Placing the Software].
+
+*Option 2*: Building a Repository Server
+
+If you need to build a separate server to act as the repository server, install a new RHEL7 system with at least 110GB of space. On this repository server during the installation make sure you select the “Basic Web Server” option.
+
+[[disconnected-placing-the-software]]
+=== Placing the Software
+
+. If necessary, attach the external storage, and then copy the repository files into Apache’s root folder. Note that the below copy step (`cp`) should be substituted with move (`mv`) if you are repurposing the server you used to sync.
++
+----
+# cp -a /path/to/repos /var/www/html/
+# chmod -R +r /var/www/html/repos
+# restorecon -vR /var/www/html
+----
```

Comment:
On my system, `cp -a` exists, but `mv -a` does not.   Suggest: "below copy step (`cp -a`) should be".